docs/output.md

```diff
@@ -41,6 +41,7 @@ This directory contains the zipped results. When unzipping them, the following s
   - `boxplots_example_genes/`: boxplots of the normalized gene counts for each of the sample groups for example genes.
   - `boxplots_requested_genes/`: boxplots of the normalized gene counts for each of the sample groups for the list of requested genes.
   - `further_diagnostics_plots/`: plots for diagnostics of the differential gene analysis procedure.
+  - `volcano_plots/`: volcano plots with differentially expressed features. Output comprises static (`.png`) and interactive plots (`.html`).
```

Comment:
do those interactive plots require their folders (which arent mentioned here) or are those self-contained html?

Reply:
The html plots are self-contained and do not require their output folders

Reply:
Nice!